### PR TITLE
add workload::Region::R4 to differentiate from workload::Region::UNKNOWN

### PR DESCRIPTION
**Pull request by AndrewMeadows**
This PR has experimental unfinished work to address the problem with avatars falling through the floor: BUGZ-401:

https://highfidelity.atlassian.net/browse/BUGZ-401

I'm posting this publicly to help other devs workaround an assert that is blocking their work.

### CODE REVIEW

File: interface/src/avatar/OtherAvatar.cpp [hunk 1]
```diff
@@ -219,7 +220,7 @@ bool OtherAvatar::isInPhysicsSimulation() const {
 }
 
 bool OtherAvatar::shouldBeInPhysicsSimulation() const {
-    return !isDead() && _workloadRegion <= workload::Region::R3;
+    return !isDead() && _workloadRegion < workload::Region::R3;
```
Comment (SamGondelman) [inline interface/src/avatar/OtherAvatar.cpp]: I guess this would be...`_workloadRegion <= workload::Region::R4` now?

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19845/HighFidelity-Beta-PR15707-e80410a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19845/HighFidelity-Quest-Beta-PR15707-e80410a.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19845/HighFidelity-Beta-PR15707-e80410a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19845/HighFidelity-Beta-Interface-PR15707-e80410a.exe

**Comment by AndrewMeadows** [comment]
Jenkins, rebuild this please.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19863/HighFidelity-Beta-PR15707-e80410a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19863/HighFidelity-Beta-Interface-PR15707-e80410a.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19863/HighFidelity-Beta-PR15707-e80410a.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19863/HighFidelity-Quest-Beta-PR15707-e80410a.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19863/HighFidelity-Beta-PR15707-e80410a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19863/HighFidelity-Beta-Interface-PR15707-e80410a.exe

**Comment by AndrewMeadows** [comment]
Not going to merge this.

**Pull request closed without merging**